src/ethsign/ethsign.go

```diff
@@ -41,16 +41,13 @@ func rawSignHash(data []byte) []byte {
   return crypto.Keccak256([]byte(msg))
 }
 
-func getWallets(c *cli.Context, defaultKeyStores cli.StringSlice) []accounts.Wallet {
+func getWallets(c *cli.Context, defaultKeyStore string) []accounts.Wallet {
   backends := []accounts.Backend{}
 
-  paths := c.StringSlice("key-store")
-
-  for _, x := range(paths) {
-    ks := keystore.NewKeyStore(
-      x, keystore.StandardScryptN, keystore.StandardScryptP)
-    backends = append(backends, ks)
-  }
+  path := c.String("key-store")
+  ks := keystore.NewKeyStore(
+      path, keystore.StandardScryptN, keystore.StandardScryptP)
+      backends = append(backends, ks)
```

Comment:
nitpick: alignment.

```suggestion
    path, keystore.StandardScryptN, keystore.StandardScryptP)
  backends = append(backends, ks)
```

Reply:
The whole file needs a `gofmt`, so figured i'd avoid doing it myself